vimeo-networking/src/main/java/com/vimeo/networking/Search.java

```diff
@@ -200,36 +199,40 @@ public static Facet fromString(String text) {
         }
     }
 
-    static final String FILTER_CATEGORY = "filter_category";
-    static final String FILTER_UPLOADED = "filter_uploaded";
-    static final String FILTER_DURATION = "filter_duration";
-    static final String FILTER_TYPE = "filter_type";
-    static final String FILTER_FEATURED_COUNT = "featured_clip_count";
-    static final String PARAMETER_GET_FACETS = "facets";
+    private static final String FILTER_CATEGORY = "filter_category";
+    private static final String FILTER_UPLOADED = "filter_uploaded";
+    private static final String FILTER_DURATION = "filter_duration";
+    private static final String FILTER_TYPE = "filter_type";
+    private static final String FILTER_FEATURED_COUNT = "featured_clip_count";
+    private static final String PARAMETER_GET_FACETS = "facets";
 
     public static Call<SearchResponse> search(@NotNull String query, @NotNull FilterType type,
-                                              @NotNull ModelCallback<SearchResponse> callback,
+                                              @NotNull VimeoCallback<SearchResponse> callback,
```

Comment:
Can we move all callbacks to become the last parameter in the method to allow for in inlining the callback? You can use the refactoring tools in android studio to do this automatically. I would suggest that all methods in VimeoClient that take a callback should take the callback as their last parameter.